libs/cardano-ledger-core/src/Cardano/Ledger/Plutus/CostModels.hs

```diff
@@ -162,14 +162,31 @@ parseCostModel o lang = do
       _ -> fail $ "Expected either an Array or an Object, but got: " ++ show plutusCostModelValue
 
 costModelFromMap :: MonadFail m => Language -> Map Text Int64 -> m CostModel
-costModelFromMap lang cmMap =
-  mapM lookupFail paramNames >>= validateCostModel lang
+costModelFromMap lang' cmMap = do
```

Comment:
Would you mind leaving this as `lang` and maybe `l` for the one in `reportMissing` ?  
```suggestion
costModelFromMap lang cmMap = do
```

Reply:
Will do, I'll make the change tomorrow :+1: .

Reply:
@teodanciu done!

Reply:
Thanks!